flake.nix

```diff
@@ -147,6 +145,16 @@
             # SITE_STYLES skipped by design.
             export NIX_PILLS_MANUAL_IN="${nixPills}/share/doc/nix-pills"
             export NIX_DEV_MANUAL_IN="${nix-dev.defaultPackage.x86_64-linux}/html"
+
```

Comment:
Since we're using the shell hook, could we provide a `serve` alias that wraps the method by which the site is served, so that whenever it is changed in the future, users don't have to care about the implementation detail?

"Whenever?" you ask? We don't know what tooling is going to be used for the website once it's migrated to a better integrated markdown tooling following that RFC. That tooling might provide its own method of live reloading.

Reply:
:+1: awesome idea.